src/ConstrainedPolyModel.cc

```diff
@@ -98,13 +98,13 @@ reference exposure, expect troubles" << std::endl;
 	  _chipMap[chip] = std::unique_ptr<SimplePolyMapping>( new SimplePolyMapping(norm,
 										     GtransfoPoly(degree)));
 	}
-      _mappings[&im] = std::unique_ptr<TwoTransfoMapping>(new TwoTransfoMapping(_chipMap[chip].get(), _shootMap[shoot].get()));
+      _mappings[&im] = std::unique_ptr<TwoTransfoMapping>(new TwoTransfoMapping(_chipMap[chip].get(), _visitMap[visit].get()));
 
     }
   cout << "INFO: ConstrainedPolyModel : we have " << _chipMap.size() << " chip mappings " << endl;
-  cout << "INFO: and " << _shootMap.size() << " shoot mappings " << endl;
+  cout << "INFO: and " << _visitMap.size() << " visit mappings " << endl;
   // DEBUG
-  for (auto i=_shootMap.begin(); i != _shootMap.end(); ++i)
+  for (auto i=_visitMap.begin(); i != _visitMap.end(); ++i)
```

Comment:
Consider using a range-based for loop here and elsewhere, though that might be out of scope.